### PR TITLE
add outputs section with pvlib and pvlib-python

### LINKED ISSUE #30: create pvlib alias
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

If I want to install pvlib-python from pypi then I run

``pip install pvlib``

and from the possibly soon to be deprecated pvlib channel I run

``conda install pvlib -c pvlib``

but if I want to use conda-forge then I have to run

``conda install pvlib-python -c conda-forge``

That inconsistency of `pvlib` vs `pvlib-python` is annoying. It's especially annoying when I have a requirements.txt file that specifies pvlib and I can't use ``conda create -n test --file requirements.txt -c conda-forge``. 

I would like to alias `pvlib` to `pvlib-python` so that either works. The kind folks in the conda-forge [gitter channel](https://gitter.im/conda-forge/conda-forge.github.io?at=6297d127f8daa71e07ab7570) pointed out that you can specify additional blocks in the ``output`` section of ``recipe/meta.yaml`` e.g. [here](https://github.com/conda-forge/importlib_metadata-feedstock/blob/main/recipe/meta.yaml#L15) and [here](https://github.com/conda-forge/typing_extensions-feedstock/blob/main/recipe/meta.yaml#L16). conda output docs [here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#outputs-section)

Historical footnote: I initially set up the conda forge name on accident but then left it alone after thinking maybe pvlib-matlab or some future pvlib-r project might be created.

cc @mikofski @kanderso-nrel @AdamRJensen @cwhanse 

### Installed packages

```shell
n/a
```

### Environment info

```shell
n/a
```

### PR DESCRIPTION
**Pull request by wholmgren**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* ~Reset the build number to `0` (if the version changed)~
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #30

### CODE REVIEW

**Comment by wholmgren** [comment]
It seems that the recommended {{ PYTHON }} variable for the script block doesn't work within outputs. Just using bare python seems to work though. Not sure how to easily test without merging. I guess we could nag conda forge admins or the gitter channel but not sure it's worth it.

**Comment by mikofski** [comment]
Wow! Will this create an alias package called pvlib that can be conda installed? Yes I happily approve!

**Comment by wholmgren** [comment]
That's the hope!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,46 +11,61 @@ source:
...
-test:
-  imports:
-    - pvlib
+  - name: pvlib-python
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python >=3.7
+        - pip
```
Comment (kandersolar) [inline recipe/meta.yaml]: Probably doesn't matter, but the importlib-metadata and typing-extension recipes don't include `host` requirements for the secondary output.

**Review by kandersolar — APPROVED** [review]
Neat!  Seems pretty straightforward.

File: recipe/meta.yaml (hunk 1 above)
Comment (wholmgren) [reply recipe/meta.yaml]: Good catch. Holdover from failed experiment while trying to make it work with `{{ PYTHON }}`. Better to keep it simple and remove it if doesn't actually help us.

**Comment by wholmgren** [comment]
@conda-forge-admin, please lint

**Comment by wholmgren** [comment]
It works! `conda install -c conda-forge pvlib` works as expected. There is an unexpected weird thing with the `pvlib-python` package... when I run

```
conda create -n pvlibcftest python=3.10 pvlib-python -c conda-forge
```

I get both `pvlib` and `pvlib-python` packages:

```
$ conda list
...
pip                       22.2.2             pyhd8ed1ab_0    conda-forge
pvlib                     0.9.2              pyha770c72_2    conda-forge
pvlib-python              0.9.2                hd8ed1ab_2    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
...
```

Maybe that *should* be expected and I was just too lazy to fully understand the `meta.yaml` and build process. 

A similar thing happens with `conda install -c conda-forge importlib_metadata` (which I partially based the recipe on)

```
$ conda list
...
idna                      3.3                pyhd8ed1ab_0    conda-forge
importlib-metadata        4.11.4          py310h2ec42d9_0    conda-forge
importlib_metadata        4.11.4               hd8ed1ab_0    conda-forge
jpeg                      9e                   hac89ed1_2    conda-forge
...
```

So I think we're good, just wanted to document it all.

**Pull request merged**